### ISSUE TITLE
Luminescence-package.R:1: @title must be a single paragraph

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#418]
I always get this message when running `devtools::check()` or `devtools::document()`:
```
ℹ Updating Luminescence documentation
ℹ Loading Luminescence
✖ Luminescence-package.R:1: @title must be a single paragraph.
```

This happens because we have a `\cr` in the title:
https://github.com/R-Lum/Luminescence/blob/5eb48d8b29d8ff78740b0e12d990da4a0dcf3ffd/R/Luminescence-package.R#L1

Can we remove it or is it somehow needed for the pdf manual or something else?

**Comment by RLumSK** [comment]
It is because to the logo of the package, otherwise it does not work. If it is only this message and we don't have another option, I would prefer to keep it.

**Comment by mcol** [comment]
Where can I see that?

**Comment by mcol** [comment]
OK, it seems that the issue is not the `\cr` but the empty lines after it. If I remove them the warning disappears and the output seems unchanged.

**Comment by mcol** [comment]
Fixed in d22fe53d312af9f754bc96f652e70ef1b9193747.

**Comment by RLumSK** [comment]
@mcol: Cool! I've this before and it did not work, but great!
